### PR TITLE
professions: add video blogger

### PR DESCRIPTION
**Pull request by sonphantrung**
#### Summary
Content "add YouTuber"

#### Purpose of change
YouTubers do exist in game

#### Describe the solution
Add YouTuber profession: Survivor but I've added laptops, message bag to backpack, and wearable camera (doubt there's a video recorder in game)

#### Describe alternatives you've considered
Not do this

#### Testing
Should be good

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
~~*Angry Twitch noises*~~

This name is bit specific, can it be broadened out so other sites can be included in it?

**Comment by nopenoperson** [comment]
they work with tech and computers all day long so adding computer and electronics makes a lot sense imho.

File: data/json/professions.json [hunk 1]
```diff
@@ -537,6 +537,22 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "vid_blogger",
+    "name": "Video Blogger",
+    "description": "You're a video blogger, you create video content and post it on various social medias.  Now most of your fans become undead, alongside your remaining equipments, you may want to see if your social skills learnt from creating content help you somehow.",
```
Comment (NetSysFire) [inline data/json/professions.json:544]:
```suggestion
    "description": "You're a video blogger, creating and commenting video content and then posting it on various platforms.  Now most of your fans are dead, alongside your equipment. Maybe leading a fanbase, moderating myriads of comments and surviving shitstorms has taught you something valuable after all.",
```

Some rewording. I feel like the original description is, no offense meant, a bit clumsy. Mine could use some improvement, too but you should see what I tried to improve.

I used "various platforms" to circumvent the awkward "social medias" mention, even though merriam webster said its both singular and plural.

File: data/json/professions.json [hunk 2]
```diff
@@ -537,6 +537,22 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "vid_blogger",
```
Comment (NetSysFire) [inline data/json/professions.json:542]:
```suggestion
    "id": "video_blogger",
```

No reason to shorten that.

**Review by NetSysFire — COMMENTED** [review]
I agree with The Shadow Ferret that they should have at least computers 1. Not too sure about electronics because you do not need to know how to build antennas to stream and/or upload content. Installing some hardware, mostly cameras, should be covered by the computer skill.

You also still need to update your PR description to reflect the YouTuber -> Video Blogger change.

**Comment by TheShadowFerret** [comment]
>I agree with The Shadow Ferret that they should have at least computers 1.

I both agree with computers 1 and am not nopenoperson, who said that. :p

**Review by MNG-cataclysm — APPROVED** [review]

**Pull request merged**